t/45bindnamedparam_error.t

```diff
@@ -0,0 +1,53 @@
+use strict;
+use warnings;
+
+use Test::More;
+use DBI;
+use vars qw($test_dsn $test_user $test_password);
+use lib 't', '.';
+require 'lib.pl';
+
+my $dbh;
+eval {$dbh = DBI->connect($test_dsn, $test_user, $test_password,
+  { RaiseError => 1, AutoCommit => 1}) or ServerError();};
+
+if ($@) {
+    plan skip_all => "no database connection";
+}
+plan tests => 11;
+
+SKIP: {
+    skip 'SET @@auto_increment_offset needs MySQL >= 5.0.2', 2 unless $dbh->{mysql_serverversion} >= 50002;
```

Comment:
DBD::mysql needs MySQL 8.0 or newer, so this shouldn't be needed. But on the other hand... it doesn't hurt either.